app/index.ts

```diff
@@ -149,7 +149,7 @@ export class Auth {
       this.setupRefreshTimer(token.expires_in);
       this.getUser(token.access_token, (response: any) => {
         let user = response.data;
-        window.location.href = '/' + user.attributes.username;
+        window.location.href = '/_home';
```

Comment:
This works, but given we know the URL, we can remove the call to getUser() entirely and just redirect synchronously after the refreshtimer.